src/vmm_mad/exec/one_vmm_exec.rb

```diff
@@ -1072,6 +1072,15 @@ def detach_nic(id, drv_message)
                     :action       => :clean
                 }
             ]
+        elsif nic_alias
+            steps = [
+                # Execute post-boot networking setup
+                {
+                    :driver       => :vnm,
+                    :action       => :post,
```

Comment:
Why not clean action?

Reply:
Clean action delete the whole iptables chain included the rules for the parent NIC